src/style.scss

```diff
@@ -0,0 +1,104 @@
+$color-blue-accent: #00ACDC;
+
+body {
+  margin: 0;
+  font-family: Roboto;
+  font-size: 14px;
+}
+
+.card {
+  height: 406px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+ &__image {
+  width: 100%;
+ }
+ &__button {
+  border: none;
```

Comment:
look the instruction we should add transparent border, and use border color on hover